### PR TITLE
Move sidebar footer styles in layout scss

### PR DESCRIPTION
**Pull request by matteodepalo**

### CODE REVIEW

File: components/_layout.scss [hunk 1]
```diff
@@ -196,8 +196,43 @@ $include-html-paint-layout: true !default;
         height: $layout-sidebar-footer-height;
         left: 0;
         line-height: $layout-sidebar-footer-height;
+        padding: 0 $layout-sidebar-nav-padding;
         position: absolute;
         right: 0;
+        white-space: nowrap;
+
+        .picture {
```
Comment (deioo) [inline components/_layout.scss]: This could be `.avatar` to keep everything consistent.
Also the `.name` below could become `.title`.

**Comment by deioo** [comment]
I'm thinking maybe we would be better to have this bit as a mixin to separate it from the sidebar wireframe logic.
What I see is something like 

```scss
@mixin sidebar-footer-with-actions($toggle-actions-icon: cog, $collapsed-styles-only: false) {
  $avatar-selector: '.avatar';
  $title-selector: '.title';
  $actions-selector: '.toggle-actions';

  @if !$collapsed-styles-only {
    #{$avatar-selector} {
      float: left;
      margin-right: $layout-sidebar-nav-padding;

      img {
        border-radius: 50%;
        height: $layout-sidebar-nav-link-icon-width;
        width: $layout-sidebar-nav-link-icon-width;
      }
    }

    #{$title-selector} {
      color: $layout-sidebar-nav-link-text-color;
      margin-right: $layout-sidebar-nav-padding / 2;
      overflow: hidden;
      text-overflow: ellipsis;
    }

    #{$actions-selector} {
      display: inline-block;
      float: right;
      line-height: $layout-sidebar-footer-height;

      button {
        @include button-icon($toggle-actions-icon);

        background-color: transparent;
        border: 0;
        color: $layout-sidebar-nav-link-text-color;
        margin-bottom: 0;
      }
    }
  } @else {
    #{$avatar-selector},
    #{$title-selector} {
      display: none;
    }

    #{$actions-selector} {
      display: block;
      float: none;
      text-align: center;
    }
  }
}
```

and then call it like
```scss
    @if $layout-sidebar-include-footer {
      #{$layout-sidebar-footer-selector} {
        @include sidebar-footer-with-actions;
         ...
```

then for the collapsed side
```scss
      @if $layout-sidebar-include-footer {
        #{$layout-sidebar-footer-selector} {
          @include sidebar-footer-with-actions($collapsed-styles-only: true);
        }
      }
```

**Comment by deioo** [comment]
The reason behind abstracting this would be the fact that e.g we might want different actions within the footer, e.g just a sign-out link.
All the selectors were declared inside the mixins as variables because they're called twice in the code.

**Comment by matteodepalo** [comment]
Yeah I like it! Maybe having 2 mixins for the collapsed and not collapsed styles would make it clearer.

**Comment by matteodepalo** [comment]
:+1:

**Comment by deioo** [comment]
:+1:

image

**Comment by toreriklinnerud** [comment]
Haha

**Pull request merged**